### PR TITLE
First take the the photon colors readme

### PR DESCRIPTION
**Pull request by aminalhazwani**
@bwinton can you help me in giving a general "how to" for `.js`, `.xml` and `.swift`? Also feel free to leave comments if you see odd things. 

Just be aware, this is very much work in progress 🙂

### CODE REVIEW

File: photon-colors/README.md [hunk 1]
```diff
@@ -1,7 +1,107 @@
...
+```xml
+<item android:color="@color/blue_60" />
+```
+
+**Swift**
+
+Download and paste [photon-colors.android.xml](#) to your Swift project. Then:
+
+```swift
+titleText.textColor = UIColor.Blue60
```
Comment (bwinton) [inline photon-colors/README.md]: This should be `UIColor.Photon.Blue60`.

File: photon-colors/README.md [hunk 2]
```diff
@@ -1,7 +1,107 @@
...
+
+.class {
+  color: @blue-60;
+}
+```
+
+**JS**
+
+```js
+import photon-colors from 'photon-colors.js';
```
Comment (bwinton) [inline photon-colors/README.md]:
```js
import photon from 'photon-colors.js';
document.getElementById('ok_button').style.color = photon.BLUE_60;
```

File: photon-colors/README.md [hunk 3]
```diff
@@ -1,7 +1,107 @@
...
+```
+
+**XML**
+
+Download and paste [photon-colors.android.xml](#) to your Android project. Then:
+
+```xml
+<item android:color="@color/blue_60" />
```
Comment (bwinton) [inline photon-colors/README.md]:
```java
textColorSpan = new ForegroundColorSpan(ContextCompat.getColor(context, R.color.blue_60));
final SpannableStringBuilder builder = new SpannableStringBuilder(text);
builder.setSpan(mUrlColorSpan, 0, text.length(), Spannable.SPAN_INCLUSIVE_INCLUSIVE);
```

File: photon-colors/README.md [hunk 4]
```diff
@@ -1,7 +1,107 @@
...
+```
+
+**Swift**
+
+Download and paste [photon-colors.android.xml](#) to your Swift project. Then:
```
Comment (bwinton) [inline photon-colors/README.md]: "`.android.xml`"? 😉

**Review by bwinton — APPROVED** [review]
Mostly good. Some small tweaks. 🙂

File: photon-colors/README.md (hunk 3 above)
Comment (aminalhazwani) [reply photon-colors/README.md]: Isn't there anything more simple that we can highlight? I took inspiration from this line of code https://dxr.mozilla.org/mozilla-central/source/mobile/android/app/src/photon/res/color/action_bar_menu_item_colors.xml#20. Or is `<item android:color="@color/blue_60" />` too much out of context?

**Comment by aminalhazwani** [comment]
As always thank you very much @bwinton for your review! I implemented all the requested changes. I just have a doubt regarding the android example, I find the suggestion to be a bit too complex, see [inline comment](https://github.com/FirefoxUX/design-tokens/pull/44#discussion_r169901538).

As we already experienced there is no way to add a _download_ link in the README for files in folders. For the time being I add a link to the raw version of said files. The only alternative is to place the generated files in the root folder, then it would be possible to have a download link.

File: photon-colors/README.md (hunk 3 above)
Comment (bwinton) [reply photon-colors/README.md]: Man, I dunno. Maybe ask someone who works on the Android browser? ¯\_(ツ)_/¯

Comment (aminalhazwani) [reply photon-colors/README.md]: 😂 I asked since you suggested that string. I'll touch base with android folks!

**Pull request merged**